b3lb/rest/b3lb/utils.py

```diff
@@ -14,50 +14,11 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-# This function file contains functions without import of b3lb files to prevent circular imports
-from db_file_storage.storage import DatabaseFileStorage
-from django.http import HttpResponse
-from django.db import models
-from wsgiref.util import FileWrapper
-import re
-
-storage = DatabaseFileStorage()
-forwarded_host_regex = re.compile(r'([^:]+)(:\d+)?$')
-
-
-def get_file_response_from_storage(file_name):
-    try:
-        stored_file = storage.open(file_name)
-    except models.Model.DoesNotExist:
-        return HttpResponse("Not Found", status=404)
-
-    response = HttpResponse(FileWrapper(stored_file), content_type=stored_file.mimetype)
-    response['Content-Length'] = stored_file.tell()
-
-    return response
-
-
-def get_file_from_storage(file_name):
-    try:
-        return storage.open(file_name).file.read()
-    except models.Model.DoesNotExist:
-        return None
-
-
-def get_forwarded_host(request):
-    forwarded_host = request.META.get('HTTP_X_FORWARDED_HOST', request.META.get('HTTP_HOST'))
-    return forwarded_host_regex.sub(r'\1', forwarded_host)
-
+# This utils file contains functions without import of b3lb files to prevent circular imports
 
 def xml_escape(string):
     if isinstance(string, str):
-        escape_symbols = [
-            ("&", "&amp;"),
-            ("<", "&lt;"),
-            (">", "&gt;"),
-            ("'", "&apos;"),
-            ('"', "&quot;")
-        ]
+        escape_symbols = [("&", "&amp;"), ("<", "&lt;"), (">", "&gt;"), ("'", "&apos;"), ('"', "&quot;") ]
```

Comment:
Should Python's built-in XML escaping functions be used instead?

https://wiki.python.org/moin/EscapingXml

Reply:
used built-in escaping function